.ahoy/site/build.ahoy.yml

```diff
@@ -59,11 +59,11 @@ commands:
   custom:
     usage: Downloads 3rd party modules
     cmd: |
-      if [ -f config/custom.make ] && [ `cat config/custom.make | sed '/^\s*$/d' | wc -l` -gt 3 ]; then
+      if [ -f config/custom.make ] && [ `cat config/custom.make | sed '/^\s*$/d' | wc -l` -gt 2 ]; then
```

Comment:
@dkinzer are you referring to the following when you say default?
```
api = 2
core = 7.x
```

If that is the case we can either make the condition look like `-ge 3` or `-gt 2`. This way it will not run if we have only those default two lines in the file.